mern-ecommerce/controllers/auth.js

```diff
@@ -73,10 +73,12 @@ exports.isAuth = (req, res, next) => {
 };
 
 exports.isAdmin = (req, res, next) => {
-  if (req.profile.role === 0) {
-    return res.status(403).json({
-      error: 'Admin resource! Access denied',
+  if (req.profile.role == 0) { 
+    res.status(403); 
+    res.json({
+      error: 'Admin resource! Access denied'
```

Comment:
The error message 'Admin resource! Access denied' is misleading and should be changed to 'Access denied. Admin privileges required.' to accurately convey the reason for access denial.